packages/nodes-base/nodes/SpreadsheetFile/v2/SpreadsheetFileV2.node.ts

```diff
@@ -92,6 +92,13 @@ export class SpreadsheetFileV2 implements INodeType {
 							to: maxRowCount > -1 ? maxRowCount : undefined,
 							columns: options.headerRow !== false,
 							onRecord: (record) => {
+								if (!options.includeEmptyCells) {
+									record = Object.fromEntries(
+										Object.entries(record).filter(
+											([_key, value]) => value !== undefined && value !== null && value !== '',
```

Comment:
I don't think it is possible but I also didn't want to take a chance on it, I could change this to just `''` and we can add the others if they do somehow appear it would be a quick change to add them back.

Reply:
@netroy changed to only use `''`